features/comment_focused_document.feature

```diff
@@ -0,0 +1,16 @@
+#language: fr
+
+Fonctionnalité: Créer un commentaire à partir d'un fragment de document en cours de focus
+
+Scénario: Une glose dont je suis l'auteur est ouverte
+
+  Soit "Treignes, le 8 septembre 2012 (Christophe Lejeune)" le document principal
+  Et une session active avec mon compte
+  Et un document dont je suis l'auteur affiché comme glose
+  Quand je sélectionne le fragment de texte :
+    """
+    plusieurs personnes se présentent à moi. Ayant identifié que je suis nouveau, elles me souhaitent la bienvenue
+    """
+  Alors je peux créer un commentaire
+  Et je peux lire "plusieurs personnes se présentent à moi. Ayant identifié que je suis nouveau, elles me souhaitent la bienvenue:"
```

Comment:
You should use a multiline string for this because the markdown format says that you should have a new line before `:`.
Moreover I think that you should put `<TEXT>` in the generated content so that people understand that you have to put your comments just after `:`.

Reply:
Great suggestion, text is added in commit https://github.com/Hypertopic/HyperGlosae/commit/51d41dbb74a7547abdb8a8570e42814da4d3af43